README.md

```diff
@@ -45,23 +47,40 @@ To grant access:
 Clone the repo to your development machine. 
 
 The project has two relevant Nuget packages:
-1. Microsoft.Azure.Services.AppAuthentication - makes it easy to fetch access tokens for Service-to-Azure-Service authentication scenarios. 
-2. Microsoft.Azure.ServiceBus - contains methods for interacting with Service Bus. 
+1. Azure.Identity - makes it easy to fetch access tokens for Service-to-Azure-Service authentication scenarios.
+2. Azure.Messaging.ServiceBus - contains methods for interacting with Service Bus. 
+
+```C# Snippet:ServiceBusAuthConnString
+// Create a ServiceBusClient that will authenticate using MSI.
+string nameSpace = "<service_bus_namespace>";
+ServiceBusClient client = new ServiceBusClient(nameSpace, new DefaultAzureCredential());
+
+// create the sender
+string queueName = "<service_bus_queueName>";
+ServiceBusSender sender = client.CreateSender(queueName);
 
-The relevant code is in WebAppServiceBus/WebAppServiceBus/Controllers/HomeController.cs file. The **Send** method will create an instance of a Service Bus QueueClient, configured to use a ManagedServiceIdentityTokenProvider. This TokenProvider's implementation uses the AzureServiceTokenProvider found in the Microsoft.Azure.Services.AppAuthentication library. AzureServiceTokenProvider will follow a set number of different methods, depending on the environment, to get an access token. Please refer to the AppAuthentication library's documentation for [local development authentication](https://docs.microsoft.com/en-us/azure/key-vault/service-to-service-authentication#local-development-authentication) to see more detail.
```

Comment:
This should be added back. References to AppAuthentication library should be replaced with Azure.Identity. AzureServiceTokenProvider should be replaced with DefaultAzureCredential.

Reply:
Updated, check it please.

Reply:
LGTM!